packages/bitcoin-rpc-provider/lib/BitcoinRpcProvider.js

```diff
@@ -35,45 +35,12 @@ export default class BitcoinRpcProvider extends JsonRpcProvider {
     }
   }
 
-  async signMessage (message, from) {
-    from = addressToString(from)
-    return this.jsonrpc('signmessage', from, message).then(result => Buffer.from(result, 'base64').toString('hex'))
-  }
-
   async sendTransaction (to, value) {
     to = addressToString(to)
     value = BigNumber(value).dividedBy(1e8).toNumber()
     return this.jsonrpc('sendtoaddress', to, value)
   }
 
-  async sendBatchTransaction (transactions) {
```

Comment:
I think @mattBlackDesign needed this for something.

Reply:
Yes, we should keep this

Reply:
Yep as discussed with @monokh the functionality of adding outputs to a `txb` in `bitcoinjs-lib` `TransactionBuilder` can be extracted to be used for both `sendTransaction` and `sendBatchTransaction`

Reply:
It was just moved below the file. With the `BitcoinLedgerProvider`, it has been refactored to remove the duplication. 
Note that the functions used with the node provider to enable this are deprecated in the latest version of Bitcoin Core and will be removed in the next version.